primary/views/manage/config/editform.pug

```diff
@@ -8,127 +8,24 @@ block content
 	p 
 		br 
 		i Modify, copy, paste from elsewhere, etc. the JSON defining your form
+		br
+		i Recommendation: Use a text editor like <a href="https://vscode.dev" class="link" target="_blank">Visual Studio Code</a> to edit the JSON, then paste it here
 	div(class="w3-container w3-section w3-auto")
 		textarea(class="w3-input w3-border" id="jsonfield" name="jsonfield" rows="40" cols="80")=layout
-		div(class="gear-btn theme-link w3-btn w3-section" onclick="test()")
+		div(class="gear-btn theme-link w3-btn w3-section" id="testBtn")
 			span Test form [HIT BACK BUTTON TO RETURN]
```

Comment:
it might be good to add a back button on that page calling [history.back()](https://developer.mozilla.org/en-US/docs/Web/API/History/back) and some sort of confirmation when closing that the form changes are unsaved and you need to go back to get them

Reply:
Or maybe instead of this being a full page load just the form (no navigation) into an iframe/popup modal to look at but not submit